common/changes/@microsoft/rush/rush-version-performance_2023-04-20-20-57.json

```diff
@@ -2,7 +2,7 @@
   "changes": [
     {
       "packageName": "@microsoft/rush",
-      "comment": "Improve performance of 'rush version' when using 'workspace:' protocol.",
```

Comment:
Why did you delete this message?

Reply:
It's redundant with https://github.com/microsoft/rushstack/blob/main/common/changes/%40microsoft/rush/rush-version-workspace-perf_2023-04-20-22-52.json, which says "Fix a performance bug in `rush version` when using `workspace:` protocol."